CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+CHANGES IN VERSION 1.9.1 (from 1.9.0)
+===================================
+
+Appium 1.9.1 is a patch release
+
+#### General
+* Append the current session identifier to the protocol name prefix
```

Comment:
this should also include multiprotocol handling fixes: https://github.com/appium/appium-base-driver/pull/258/files